### PR TITLE
EXUI-1729: Validation error messages added for Hearing location

### PR DESCRIPTION
**Pull request by SomeshRA**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/EXUI-1729

### Change description ###
Validation error messages added for Hearing location

### Checklist

- [x ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by RiteshHMCTS — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -96,7 +96,7 @@
     "@hmcts/media-viewer": "3.0.2",
     "@hmcts/nodejs-healthcheck": "1.7.0",
     "@hmcts/properties-volume": "^0.0.13",
-    "@hmcts/rpx-xui-common-lib": "2.0.17",
+    "@hmcts/rpx-xui-common-lib": "v2.0.17-search-venue-fix",
```
[the remark this answers is not in the document]
Comment (SomeshRA) [reply package.json:99]: @RiteshHMCTS this was suggested by @andywilkinshmcts

[the remark this answers is not in the document]
Comment (andywilkinshmcts) [reply package.json:99]: Some understandable confusion. The tag has v at the start, but the version number itself should not. Unfortunately you'll have to do another pre-release with tag "v2.0.17-search-venue-fix-2" but version number "2.0.17-search-venue-fix"

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]
Please update the version number for your pre-release

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by RiteshHMCTS — APPROVED** [review]

**Pull request merged**